start_elasticsearch.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Start elasticsearch in a docker container
+
+[ -z "$ES_VERSION" ] && echo "Need to set ES_VERSION (2.4, 5.3, 5.4)" && exit 1;
```

Comment:
> could we default to `5.4.0` for now?

Yes can make that the default

> better yet, is there `latest` or `5.x`

Not yet. but all I need to do is `docker tag fxdgear/elasticsearch:5.4 fxdgear/elasticsearch:latest` and then push and there will be a `latest`. 

and then I'll update the default to be `latest`